docs/page-software-operation-view.md

```diff
@@ -1,3 +1,430 @@
+                  {
+                    "type": "NONE",
+                    "key": "NONE",
+                    "value": "NONE"
+                  }
+                ]
+              }
+            }
+          ],
+          "idShort": "digitalProductPass",
+          "id": "urn:uuid:777a3f0a-6d29-4fcd-81ea-1c27c1b870cc",
+          "semanticId": {
+            "type": "ExternalReference",
+            "keys": [
+              {
+                "type": "Submodel",
+                "value": "urn:samm:io.catenax.generic.digital_product_passport:3.0.0#DigitalProductPassport"
+              }
+            ]
+          },
+          "description": [
+            {
+              "language": "en",
+              "text": "Digital Product Passport Submodel"
+            }
+          ]
+        }
+      }
+    ]
+    }
+```
+> **_NOTE:_** 
+*The BPN number is not required for the configuration of the endpoint, just **make sure that the host is pointing to the EDC Provider**.*
+
+
+### Digital Twin Registry Configuration
+
+When configuring the digital twin registry behind the EDC Provider you should follow this EDC Registration guidelines:
+
+#### Variables:
+
+| Name         | Description                                 | Example Value                                                 |
+|--------------|---------------------------------------------|---------------------------------------------------------------|
+| registryUrl  | The base url from the digital twin registry | https://materialpass.int.demo.catena-x.net/semantics/registry |
+| registryName | The name from the asset for the registry    | digital-twin-registry                                         |
+
+#### Format and Fields:
+
+```json
+{
+    "@context": {},
+    "asset": {
+        "@type": "data.core.digitalTwinRegistry",
+        "@id": "{{registryName}}", 
+        "properties": {
+            "description": "Digital Twin Registry",
+            "contenttype": "application/json" 
+        }
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "type": "HttpData",
+        "proxyPath": "true",
+        "proxyBody": "true",
+        "proxyMethod": "true",
+        "proxyQueryParams": "true",
+        "baseUrl": "{{registryUrl}}"
+    }
+}
+```
+
+## Item Relationship Service Integration
+
+For deploying and integrating the IRS (Item Relationship Service)[https://github.com/eclipse-tractusx/item-relationship-service] with the digital product pass application first deploy the [reference helm chart values](../../deployment/helm/irs) helm charts.
+
+For creating relationships between the digital twins register "singleLevelBomAsBuilt" and "singleLevelBomAsUsage" aspects which can be found here: [SingleLevelBomAsBuilt](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_usage_as_built) and [SingleLevelUsageAsBuilt](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_usage_as_built)
```

Comment:
This sentence sounds odd, like there is something missing in the end?

Reply:
I will fix it!